src/views/Dashboards/Dashboards.js

```diff
@@ -42,6 +42,8 @@ const Dashboards = ({ currentScenario, scenarioList, reports }) => {
     setValue(newValue);
   };
 
+  const dashboardTitle = DASHBOARDS_LIST_CONFIG[value].title[i18n.language];
```

Comment:
Could we add an error message or a default value (e.g. 'MISSING TITLE') in case the "title" property is missing ?
It would be helpful for non-developers users that only want a configurable webapp :)

Reply:
:+1: